### PR TITLE
Add documentation to deploy mimir to puppet

### PR DESCRIPTION
**Pull request by bubu11e**
#### What this PR does
A module is community maintained to deploy Grafana mimir using puppet. This commit is to add a documentation page on how to use it.

### CODE REVIEW

File: docs/sources/mimir/operators-guide/deploy-grafana-mimir-with-puppet/_index.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+
+# Deploy Grafana Mimir with Puppet
+
+A puppet module is community maintained to deploy Grafana mimir.
+
+## Start using this module
+
+Add this module to your Puppetfile:
+
+    mod 'ovhcloud-mimir', '1.0.0'
```
Comment (56quarters) [inline docs/sources/mimir/operators-guide/deploy-grafana-mimir-with-puppet/_index.md:21]: Does the puppet code need to be in ` ``` ` blocks?

File: docs/sources/mimir/operators-guide/deploy-grafana-mimir-with-puppet/_index.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+## Install and configure Mimir
+
+Call the main class of the module inside a puppet manifest like this:
+
+    class{'mimir': }
+
+If call like this, this module will use the default configuration parameters of Mimir and therefore start mimir in standalone mode.
```
Comment (56quarters) [inline docs/sources/mimir/operators-guide/deploy-grafana-mimir-with-puppet/_index.md:29]:
```suggestion
If called like this, this module will use the default configuration parameters of Mimir and therefore start mimir in standalone mode.
```

File: docs/sources/mimir/operators-guide/deploy-grafana-mimir-with-puppet/_index.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+  - Mimir deployment
+  - Puppet
+menuTitle: Deploy with Puppet
+title: Deploy Grafana Mimir with Puppet
+weight: 13
+---
+
+# Deploy Grafana Mimir with Puppet
+
+A puppet module is community maintained to deploy Grafana mimir.
```
Comment (56quarters) [inline docs/sources/mimir/operators-guide/deploy-grafana-mimir-with-puppet/_index.md:15]: Does using Puppet imply that Mimir will be installed from deb/rpm? If so, is that worth mentioning?

File: docs/sources/mimir/operators-guide/deploy-grafana-mimir-with-puppet/_index.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
+---
+aliases:
+  - deploying-grafana-mimir/
```
Comment (56quarters) [inline docs/sources/mimir/operators-guide/deploy-grafana-mimir-with-puppet/_index.md:3]: This is a new page so I don't think it should have an alias.

**Review by 56quarters — COMMENTED** [review]
Thanks! A few suggestions. I'll let the docs squad chime in here but this looks pretty good to me.

File: docs/sources/mimir/operators-guide/deploy-grafana-mimir-with-puppet/_index.md (hunk 1 above)
Comment (bubu11e) [reply docs/sources/mimir/operators-guide/deploy-grafana-mimir-with-puppet/_index.md]: Thanks, fixed.

File: docs/sources/mimir/operators-guide/deploy-grafana-mimir-with-puppet/_index.md (hunk 3 above)
Comment (bubu11e) [reply docs/sources/mimir/operators-guide/deploy-grafana-mimir-with-puppet/_index.md]: Indeed, it worth mentioning.

**Review by 56quarters — APPROVED** [review]

**Pull request merged**